### ISSUE TITLE
ncaught TypeError: (void 0) is not a function with import GooglChart

### ISSUE THREAD

**Issue by singhrasster** [vimalavinisha/angular2-google-chart#51]
I followed this to import GoogleChart in app.module.ts
https://www.npmjs.com/package/angular2-google-chart
But, when I import it, I get the following error:

ncaught TypeError: (void 0) is not a function
    at eval (eval at <anonymous> (app.js:5614), <anonymous>:1:9)
    at Object.<anonymous> (app.js:5614)
    at __webpack_require__ (polyfills.js:53)
    at eval (eval at <anonymous> (app.js:5062), <anonymous>:80:41)
    at Object.<anonymous> (app.js:5062)
    at __webpack_require__ (polyfills.js:53)
    at eval (eval at <anonymous> (app.js:3059), <anonymous>:4:20)
    at Object.<anonymous> (app.js:3059)
    at __webpack_require__ (polyfills.js:53)
    at Object.<anonymous> (app.js:5580)
    at __webpack_require__ (polyfills.js:53)
    at webpackJsonpCallback (polyfills.js:24)

Is there a bug in the library or I am missing something?

**Comment by Abiwax** [comment]
I have this same issue, were you able to fix it?

**Comment by cetinahmetemre** [comment]
me too take , could you solve problem?

https://stackoverflow.com/questions/45347637/ionic-2-google-charts-void-0-is-not-a-function

**Comment by singhrasster** [comment]
No, I could not find a solution to this. If you find something, please post
the solution here if possible so it can help in future.

**Comment by cetinahmetemre** [comment]
okay. you too please

**Comment by randbrown** [comment]
I'm getting this error with angular@4.1.3 and angular2-google-chart@2.3.0.  Did anyone find a fix for this?

**Comment by chorpler** [comment]
I found a fix by taking the angular2-google-chart.directive.ts file and adding it directly to a component in my project (Ionic 3.5.0 at the moment). I suspect the main problem was that it just needed to be recompiled with my version of TypeScript (2.3.4 at the moment).
